### PR TITLE
reports: Add anctartica's GSoC report

### PR DESCRIPTION
**Pull request by anctartica**
https://deploy-preview-630--coala-projects.netlify.com/gsoc/2018/statusreport/saksham

### CODE REVIEW

File: reports/saksham.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+organisation: coala
+organisation_link : https://coala.io
+project: Improve installation of coala and coala-bears
+project_link: https://summerofcode.withgoogle.com/projects/#4642118669172736
+tarball: https://rawgit.com/anctartica/GSoC-Report/master/work.tar.gz
+mentors: >
+ [Muhammad Kaisar Arkhan](https://github.com/yukiisbored) & [Matt Bodenhamer]
+ (https://github.com/mbodenhamer) & [Andrey Safonov](https://github.com/xferra)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/10
```
Comment (jayvdb) [inline reports/saksham.md]: see https://github.com/coala/projects/issues/627 and https://github.com/coala/projects/issues/615

File: reports/saksham.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+ - 34:
+   - repo: bear-install-playbook
+   - link: https://gitlab.com/coala/bear-install-playbook/commit/7dee7591927c0883269f5741102b90bfb93f84ff
+   - details: >
+      Add instructions on using playbook
+ - 35:
+   - repo: bear-install-playbook
+   - link: https://gitlab.com/coala/bear-install-playbook/commit/c82a9737c6852d09f681a7fc98a0f094dd38194b
+   - details: >
+      Run coala static code analysis
```
Comment (jayvdb) [inline reports/saksham.md]: this was an automated change? if so, discard

Comment (anctartica) [reply reports/saksham.md]: removed all the automated changes :)

File: _reports/saksham.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+project: Improve installation of coala and coala-bears
+project_link: https://summerofcode.withgoogle.com/projects/#4642118669172736
+tarball: https://rawgit.com/anctartica/GSoC-Report/master/work.tar.gz
+sha256sum: f413963583c7e18a5ccb42f21405290edcf79a0f8c61106ff21671efe0b98238
+mentors: >
+ [Muhammad Kaisar Arkhan](https://github.com/yukiisbored),
+ [Matt Bodenhamer](https://github.com/mbodenhamer),
+ [Andrey Safonov](https://github.com/xferra)
+phase:
+ - Bonding: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/10
```
Comment (jayvdb) [inline _reports/saksham.md]: indentation; use yamllint

Comment (anctartica) [reply _reports/saksham.md]: I don't get anything from yamllint on this line

File: _reports/saksham.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/SakshamBansal
```
Comment (jayvdb) [inline _reports/saksham.md]: see updates to the example.  needs date, categories, and not layout or permalink, etc.

File: _reports/saksham.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+student: Saksham Bansal
+organisation: coala
+organisation_link: https://coala.io
+report_year: 2018
+project: Improve installation of coala and coala-bears
+project_link: https://summerofcode.withgoogle.com/projects/#4642118669172736
+tarball: https://rawgit.com/anctartica/GSoC-Report/master/work.tar.gz
+sha256sum: f413963583c7e18a5ccb42f21405290edcf79a0f8c61106ff21671efe0b98238
+mentors: >
+ [Muhammad Kaisar Arkhan](https://github.com/yukiisbored),
```
Comment (jayvdb) [inline _reports/saksham.md]: yamllint.

File: _reports/saksham.md [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+      - link: https://gitter.im/anctartica
+  - OpenHub:
+      - username: anctartica
+      - link: https://www.openhub.net/accounts/anctartica
+email: sakshamb.co.17@nsit.net.in
+blog: https://medium.com/@sakshamb.co.17
+activity:
+  - 1:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/commit/ca27c58eef778fd7b3aff3b874b0e58f56e4a559
```
Comment (jayvdb) [inline _reports/saksham.md]: you can trim these to 8 character shas.
up to you

File: _reports/saksham.md [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+  - 2018
+  - StatusReport
+report_year: 2018
+date: 2018-08-12
+student: Saksham Bansal
+organisation: coala
+organisation_link: https://coala.io
+project: Improve installation of coala and coala-bears
+project_link: https://summerofcode.withgoogle.com/projects/#4642118669172736
+tarball: https://rawgit.com/anctartica/GSoC-Report/master/work.tar.gz
```
[the remark this answers is not in the document]
Comment (jayvdb) [reply _reports/saksham.md]: Also contains `.DS_Store`

File: _reports/saksham.md [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+      - repo: bear-install-playbook
+      - link: https://gitlab.com/coala/bear-install-playbook/commit/1b288844aa157ff4e8c80a3179c5d58163a2719f
+      - details: >
+         Test playbook using docker + Travic CI
+  - 34:
+      - repo: bear-install-playbook
+      - link: https://gitlab.com/coala/bear-install-playbook/commit/7dee7591927c0883269f5741102b90bfb93f84ff
+      - details: >
+         Add instructions on using playbook
+  - 35:
```
Comment (jayvdb) [inline _reports/saksham.md]: 59 patches vs 35 work activity entries.

I see some patches were ust sync' but I easily see some real work patches which are not included here

Comment (anctartica) [reply _reports/saksham.md]: the others were really very small to include

Comment (anctartica) [reply _reports/saksham.md]: alright added some more

File: _reports/saksham.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (anctartica) [reply _reports/saksham.md]: This was due to BSD tar (on mac) vs. GNU tar.
Should be fixed now

File: _reports/saksham.md (hunk 8 above)
Comment (jayvdb) [reply _reports/saksham.md]: Small is ok, as long as it isnt automated or silly.

File: _reports/saksham.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (jayvdb) [reply _reports/saksham.md]: Still has one `.DS_Store` in it, but good enough

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/630/commits/6453618318c4abf3becef3200d7ae9b6d69382c6

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**